mlibrary/pride
# Updating `eslint`

# Overview
Updating `eslint` was a breaking change. This adds the new configuration file, copied from Search. Changes have been made to a handful of `src/Pride/*.js` files as well. Old plugins have been removed.

The hope was to resolve all errors, but that would require rewriting many of the already rewritten files. This will be done at a later time.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#branch-name-of-pull-request` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#branch-name-of-pull-request"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- _How to use the feature_.
- Make sure the PR is consistent in these browsers:
